code/modules/ninja/energy_katana.dm

```diff
@@ -73,9 +73,9 @@
 	..()
 
 /obj/item/energy_katana/Destroy()
+	. = ..()
 	QDEL_NULL(spark_system)
 	QDEL_NULL(jaunt)
-	return ..()
```

Comment:
different PR for this.

Reply:
This pr was already tried, and denied because it wasn't fixing the underlying issue